packages/neuron-ui/src/widgets/Alert/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { SuccessInfo, Error as ErrorIcon } from 'widgets/Icons/icon'
+import styles from './index.module.scss'
+
+export enum AlertStatus {
+  Init = 0,
+  Success = 1,
+  Error = 2,
+}
+
+const getClassName = (status: AlertStatus) => {
+  switch (status) {
+    case AlertStatus.Init:
+      return styles.init
+    case AlertStatus.Error:
+      return styles.error
+    case AlertStatus.Success:
+      return styles.success
+    default:
+      return styles.init
+  }
+}
+
+const Alert = ({
+  status,
+  children,
+  className,
+}: {
+  status: AlertStatus
+  children?: React.ReactElement | string
+  className?: string
+}) => {
+  return (
+    <li className={`${getClassName(status)} ${className || ''} ${styles.alert}`}>
```

Comment:
How about defining `AlertStatus` with text instead of number and adding `status` as an attribute of `li`, then `li`'s style could be set based on the `status` attribute but not a `className` from `getClassName`

Reply:
Although 
```
<li className={`${styles[status]} ${className || ''} ${styles.alert}`}>
```
works, 

```
<li className={`${className || ''} ${styles.alert}`} data-status={status}>
```

is better, but just opinionated taste 🍻